examples/hubbard.jl

```diff
@@ -58,7 +58,7 @@ U = 10u"eV"
 manifold = OrbitalManifold(Ni, "3D")
 
 # Run SCF with a DFT+U setup, notice the `extra_terms` keyword argument, setting up the Hubbard +U term.
```

Comment:
It would be good to explain in the comment here (which turns into https://docs.dftk.org/dev/examples/hubbard/, remember) how to add multiple manifolds. I think we don't need to use multiple for this example, but we should make the reader aware of it.